.github/ISSUE_TEMPLATE/component-accessibility-deliverables.yaml

```diff
@@ -0,0 +1,92 @@
+name: Component accessibility deliverables
+description: Submit the accessibility portion of your component
+title: '[Component name] component – accessibility '
+labels: 'accessibility'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Complete the following deliverables for Carbon to review the accessibility portion of the [component contribution process](https://carbondesignsystem.com/contributing/component/). To see examples, view the component accessibility guidelines on [Github](https://github.com/hiraaeth/component-contribution-accessibility/wiki/component-accessibility-guidelines) or on [figma](https://www.figma.com/file/PmSOwLS3ldpSQtUf4xlaLD/component-accessibility-guidelines?type=design&node-id=434-4103&mode=design&t=qQ2DqDAiNaOLPlBC-0).
```

Comment:
Could this github link be updated to the wiki of this repo instead of `hiraaeth/component-contribution-accessibility`? I'm happy to help copy and paste it over if that makes sense.